cpp/examples/bag2mcap.cpp

```diff
@@ -30,8 +30,8 @@ int main() {
   mcap::Channel topic("/chatter", "ros1", stdMsgsString.id);
   writer.addChannel(topic);
 
-  std::array<std::byte, 4 + 13> payload;
-  const uint32_t length = 13;
+  std::array<std::byte, 4 + 13> payload{};
```

Comment:
does this line change fix a warning?

Reply:
> does this line change fix a warning?

Specifically, this is not a warning, but according to the [cpp core guidelines](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Pro-type-memberinit) recommendation: Don't use a variable before it has been initialized